src/routes.js

```diff
@@ -53,6 +57,31 @@ function validateSignedTransactionReq({ signedTx } = {}) {
   return true
 }
 
+/**
+ * Function to ensure account request is not empty
+ * @param {Account} account
+ */
+function validateAccount({ account } = {}) {
+  if (!account) {
+    throw new Error('Account is empty.')
+  }
+}
+
+/**
+ * Checks for existence of group addresses which should not be explicitly requested
+ * @param {*} db Database
+ * @param {*} Server Server Config Object
+ */
+const checkForGroupAddress = (dbApi: DbApi, { logger }: ServerConfig) => async (
```

Comment:
technically we don't need this, frontend should ensure it is sending correct requests :-)